internal/indexmanagement/reconcile.go

```diff
@@ -338,6 +339,7 @@ func newCronJob(clusterName, namespace, name, schedule, script string, nodeSelec
 			SuccessfulJobsHistoryLimit: jobHistoryLimitSuccess,
 			FailedJobsHistoryLimit:     jobHistoryLimitFailed,
 			Schedule:                   schedule,
+			Suspend:                    pointer.BoolPtr(suspend),
```

Comment:
can't we just do `&suspend` ? Not sure why we're pulling in a new library here

Reply:
Although `"k8s.io/utils/pointer"` is not a new library, I agree for bool we can do it easier.